### PR TITLE
Update proxy server configuration

### PR DESCRIPTION
**Pull request by MaksimDell**
# Description
This PR fixes the arguments in karavictl commands.
image
image

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/583

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/authorization/configuration/proxy-server/_index.md [hunk 1]
```diff
@@ -24,30 +24,33 @@ A `storage` entity in CSM Authorization consists of the storage type (PowerFlex,
...
 
 # Helm Deployment
-karavictl storage create --type powerflex --endpoint https://10.0.0.1 --system-id ${systemID} --user ${user} --password ${password} --insecure --array-insecure --addr storage.csm-authorization.com:<ingress-nginx-controller-port>
+karavictl storage create --type powerflex --endpoint ${powerflexIP} --system-id ${systemID} --user ${user} --password ${password} --insecure --array-insecure --addr storage.csm-authorization.com:<ingress-nginx-controller-port>
 ```
 
 >__Note__:
 > - The `insecure` flag specifies to skip certificate validation when connecting to the CSM Authorization storage service.
 > - The `array-insecure` flag specifies to skip certificate validation when proxy-service connects to the backend storage array. Run `karavictl storage create --help` for help.
+> - The `powerflexIP` is the endpoint to you powerflex machine. You could find the `systemID` at the `https://<your_powerflex_ip_address>/dashboard/performance` near the `System` title.
```
Comment (rsedlock1958) [inline content/docs/authorization/configuration/proxy-server/_index.md:36]: Change "you" to "your". Change "could" to "can".

**Review by rsedlock1958 — COMMENTED** [review]
I have reviewed this. Two minor edits are required.

**Review by bjiang27 — APPROVED** [review]

**Review by bjiang27 — DISMISSED** [review]

File: content/docs/authorization/configuration/proxy-server/_index.md [hunk 2]
```diff
@@ -24,30 +24,33 @@ A `storage` entity in CSM Authorization consists of the storage type (PowerFlex,
...
+> - The `user` and `password` arguments are credentials to the powerflex UI. 
 
 ### Configuring Tenants
 
 A `tenant` is a Kubernetes cluster that a role will be bound to. For example, to create a tenant named `Finance`:
 
 ```yaml
 # RPM Deployment
-karavictl tenant create --name Finance --insecure --addr grpc.DNS-hostname:443
+karavictl tenant create --name Finance --insecure --addr grpc.<DNS-hostname>:443
```
Comment (rajkumar-palani) [inline content/docs/authorization/configuration/proxy-server/_index.md:45]: can you please check <> brackets in the local preview?

**Review by rajkumar-palani — CHANGES_REQUESTED** [review]
please share the screenshot of the local preview for the changes done.

File: content/docs/authorization/configuration/proxy-server/_index.md (hunk 2 above)
Comment (MaksimDell) [reply content/docs/authorization/configuration/proxy-server/_index.md:45]: dns

**Review by rajkumar-palani — APPROVED** [review]
lgtm

**Review by rajkumar-palani — CHANGES_REQUESTED** [review]
can you please check whether changed files are signed committed.

**Review by rajkumar-palani — APPROVED** [review]
lgtm.

**Review by rajkumar-palani — CHANGES_REQUESTED** [review]
please check whether changes are signed committed. 
image

**Review by rajkumar-palani — APPROVED** [review]
lgtm. checked the verified commit.

**Comment by rajkumar-palani** [comment]
@MaksimDell - can you please update the GITHUB link in the description.
image

**Review by bjiang27 — APPROVED** [review]

**Review by shanmydell — DISMISSED** [review]
< angular brackets will make text invisible in hugo

**Comment by MaksimDell** [comment]
> 
image
I checked this locally, it seems like that working correct. If I am wrong, what do you think the best solution would be for this argument?

**Review by MaksimDell — COMMENTED** [review]
image
image

**Comment by MaksimDell** [comment]
image
image

**Comment by MaksimDell** [comment]
> < angular brackets will make text invisible in hugo
@shanmydell
It won't, I checked this locally.
image
image

**Comment by prablr79** [comment]
Lgtm

**Pull request merged**